javascript/javascript1/week3/exercises.js

```diff
@@ -0,0 +1,267 @@
+// Step 1 : @sergii-borodin
+
+// Step 2: Exercises
+
+// Item array removal
+// Remove the item in names that is equal to nameToRemove
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+function changeNames(name) {
+  for (let i = 0; i < names.length; i++) {
+    const element = names[i];
+    if (element === name) {
+      names.splice(1, 1);
+    }
```

Comment:
on line 24,
names.splice(i,1) 
as it is now, you are always removing the first element :)